Userland/Demos/CatDog/CatDog.cpp

```diff
@@ -118,6 +118,11 @@ void CatDog::timer_event(Core::TimerEvent&)
     if (has_flag(m_state, State::Alert))
         return;
 
+    ScopeGuard update_animation_frame = [&] {
```

Comment:
commit description: https://github.com/SerenityOS/serenity/blob/master/CONTRIBUTING.md "*Don't*: Use weasel-words like "refactor" or "fix" to avoid explaining what's being changed."

Maybe a couple more words on what was wrong that's now better?

Reply:
Done :+1: